src/data/instant-observability-categories.json

```diff
@@ -90,5 +90,14 @@
     "associatedKeywords": [
       "mlops"
     ]
+  },
+  {
+    "value": "notifications",
+    "displayName": "Notifications",
+    "associatedKeywords": [
+      "notifications",
+      "outbound integrations",
+      "destinations"
+    ]
```

Comment:
Per the thread, shouldn't this just be `notifications`?

Reply:
@jerelmiller We were looking for confirmation from what is going to be defined internally, so if that is the case, do you mean the only associated keyword is `notifications` and the others should be removed?

Reply:
@jpvajda correct according to [this message](https://newrelic.slack.com/archives/C01R2TB8BUZ/p1637160342242100?thread_ts=1636378774.153100&cid=C01R2TB8BUZ)

Reply:
👍